genanki/model.py

```diff
@@ -4,13 +4,30 @@
 import yaml
 
 class Model:
-  def __init__(self, model_id=None, name=None, fields=None, templates=None, css=''):
+  def __init__(self, model_id=None, name=None, fields=None, templates=None, css='',
+          latex_header=None, latex_footer=None):
     self.model_id = model_id
     self.name = name
     self.set_fields(fields)
     self.set_templates(templates)
     self.css = css
 
+    if latex_header is None:
+        self.latex_header = """\\documentclass[12pt]{article}
```

Comment:
Could you pull this out into a constant at the top of the file? e.g. `_DEFAULT_LATEX_HEADER`.